tests/test_dse.py

```diff
@@ -2657,11 +2672,14 @@ def test_fullopt(self):
         bns, _ = assert_blocking(op1, {'x0_blk0'})  # due to loop blocking
 
         assert summary0[('section0', None)].ops == 50
-        assert summary0[('section1', None)].ops == 148
+        assert summary0[('section1', None)].ops == 44
         assert np.isclose(summary0[('section0', None)].oi, 2.851, atol=0.001)
 
-        assert summary1[('section0', None)].ops == 31
-        assert np.isclose(summary1[('section0', None)].oi, 1.767, atol=0.001)
+        assert summary1[('section0', None)].ops == 9
+        assert summary1[('section1', None)].ops == 9
+        assert summary1[('section2', None)].ops == 31
+        assert summary1[('section3', None)].ops == 26
```

Comment:
why the new sections?

Reply:
I should maybe just run the code. I will, later on

Reply:
There is no one section per sparse coord computing temps for the position

Reply:
Same as above
```
  #pragma omp parallel num_threads(nthreads_nonaffine)
  {
    int chunk_size = (int)(fmax(1, (1.0F/3.0F)*(p_sf_M - p_sf_m + 1)/nthreads_nonaffine));
    #pragma omp for schedule(dynamic,chunk_size)
    for (int p_sf = p_sf_m; p_sf <= p_sf_M; p_sf += 1)
    {
      r1[p_sf] = floor(r3*(-o_x + sf_coords[p_sf][0]));
      r2[p_sf] = floor(r4*(-o_y + sf_coords[p_sf][1]));
    }
  }
```